src/Sylius/Bundle/CoreBundle/spec/EventListener/OrderCurrencyListenerSpec.php

```diff
@@ -44,12 +46,20 @@ function it_throws_exception_if_event_has_non_order_subject(
     function it_sets_currency_on_order(
         $currencyContext,
         GenericEvent $event,
-        OrderInterface $order
+        OrderInterface $order,
+        EntityRepository $currencyRepository,
+        Currency $currency
     )
     {
         $event->getSubject()->willReturn($order);
         $currencyContext->getCurrency()->shouldBeCalled()->willReturn('PLN');
 
+        $currencyRepository->findOneBy(['code' => 'PLN'])->shouldBeCalled()->willReturn($currency);
+
+        $currency->getCode()->willReturn('EUR');
+
```

Comment:
lots of stofs here. Look at other specs but I think there will just be 1 empty line before the actual subject of the test (line 61) otherwise they are all together.